data/debug-cli/docker_debug.yaml

```diff
@@ -0,0 +1,166 @@
+command: docker debug
+short: Get a shell into any container or image.
+long: |-
+  > **Note**
+  >
+  > Docker Debug requires a [Pro, Team, or Business subcription](/subscription/details/).
+  > You must [sign in](/desktop/get-started/) to use this command.
+
+  Get an enhanced shell with additional tools into any container or image.
```

Comment:
We should explain a bit more. It's important to somehow introduce the concept of the toolbox here: 

The users can get a shell into any container/image. But they can also bring all the tools with them in that shell. Ie, the debug shell comes with a toolbox. However, this toolbox is always kept separated from the actual container/image. At no point the actually image/container will be affected/modified by that or see any of the tools. Eg, the user can run `install nmap` in the debug shell to add `nmap`. Than `nmap` can be used in the debug shell. But the actual container/image stays the same. If the user debugs something else, `nmap` can still be used because it was added to the toolbox. By default, the toolbox is pre-filled with common linux tools, eg `vim`, `nano`, `htop`..

cc @ipedrazas maybe you have something more catchy to explain this